### PR TITLE
(GH-62) Added new alias TfsVotePullRequest

### PR DESCRIPTION
**Pull request by stffabi**
Added new cake alias TfsVotePullRequest, which makes it possible to vote for the pull request. 

Fixes GH-62

### CODE REVIEW

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs [hunk 1]
```diff
@@ -98,6 +106,34 @@ public TfsPullRequestSystem(ICakeLog log, TfsPullRequestSettings settings)
...
+            {
+                var request =
+                    gitClient.CreatePullRequestReviewerAsync(
+                        new IdentityRefWithVote() { Vote = (short)vote },
+                        this.pullRequest.Repository.Id,
+                        this.pullRequest.PullRequestId,
+                        authorizedIdenity.Id.ToString(),
+                        CancellationToken.None);
+
+                var createdReviewer = request.Result;
```
Comment (pascalberger) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs]: Do we need to handle errors here?

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystemAliases.cs [hunk 2]
```diff
@@ -239,5 +239,43 @@ public static class TfsPullRequestSystemAliases
...
+        /// using the specified settings.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">Settings for accessing the pull request system.</param>
+        /// <param name="vote">The vote for the pull request.</param>
+        /// <example>
+        /// <para>Vote 'Approved' to a TFS pull request:</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var repoRoot = new DirectoryPath("c:\repo");
```
Comment (pascalberger) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystemAliases.cs]: Not required

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestVote.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+namespace Cake.Prca.PullRequests.Tfs
+{
+    /// <summary>
+    /// Defines the vote for a TFS pull request.
```
Comment (pascalberger) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestVote.cs]: Team Foundation Server or Visual Studio Team Services

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystemAliases.cs (hunk 2 above)
Comment (stffabi) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystemAliases.cs]: Fixed.

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestVote.cs (hunk 3 above)
Comment (stffabi) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestVote.cs]: Fixed.

File: src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs (hunk 1 above)
Comment (stffabi) [inline src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs]: I think for the time being it's okey to throw errors. The caller could handle them, furthermore it's consistent with the current overall implementation.

**Pull request merged**